src/server/helpers/mailer.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2021  Divyanshu Raj
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+import config from '../../common/helpers/config.js';
+import nodemailer from 'nodemailer';
+
+const {mailConfig} = config;
+
+/**
+ * This is a dedicated mailing interface for bookbrainz mailing services
+ *
+ * @param {string} from - Email address of sender
+ * @param {string} to - Email address of receiver 
+ * @param {string} subject- Subject of the E-mail
+ * @param {string} html - Body of E-mail in form of HTML 
+ * @returns {object} an object containing the error message if any error will occur.
```

Comment:
The function returns a promise, not an object.
However it is useful to also detail what that promise returns, as you've done.
```suggestion
 * @returns {Promise<object>} Returns a promise that resolves with an error object if message failed or an info object if the message succeeded (see https://nodemailer.com/usage/ for more details)
```